doc/user_guide/bank/essentials_calendar.md

```diff
@@ -1 +1,35 @@
-Not yet written
\ No newline at end of file
+# Calendar
+
+The calendar shows the Pickup and delivery schedule. It shows past and upcoming distributions in a given month.
+
+Click on the "Pick Ups & Deliveries" section to view the scheduled calendar events. 
+
+![PickUp & Delivery](images\pickup&delivery.jpg)
+
+Click on a specific event to view more details about the scheduled event. 
+
+The Distribution Schedule page shows details including the associated Partner,the Time of Distribution, Source Inventory, the Total number of items, and the status of the distribution.
+
+![Specific Day Distribution](images\specific_day_distribution_schedule.jpg)
+
+If the distribution has been done, you can mark it as complete by clicking on "Distribution Complete".
+
+Click "View" for more details on the Source location, the Agency representative, the delivery method, the Shipping cost, comments and the State of the distribution.
+
+![Distribution from Source Inventory to Partner](images\distribution_from_source_to_partner.jpg)
+ 
+Click "Print" to print the details of the distribution, for instance if you need to provide the list for those in the warehouse to pack up. 
+
+## Sync Calendar to Google Calendar
+
+You can sync the App calendar to Google calendar for more convinience.
+
+Click on "Copy Calendar URL to clipboard" to get the Calendar URL to be added to Google Calendar. On Google calendar, in the "Other Calendars" section click on the "+" which will bring up several choices.
```

Comment:
the second "Calendar" should be lower case.  The one in 'Google calendar' should be upper case in both cases (because Google Calendar is the proper name of the application.)